### ISSUE TITLE
use a cc license?

### ISSUE THREAD

**Issue by aoloe** [scribusproject/scribus-tutorials#1]
if you plan to put content and not code in this repository, it's imo better to use a cc license (cc-by-sa?) instead of a gpl one.

gpl is for code...

(or, at least, double license it...)

**Comment by OceanWolf** [comment]
What kind of content do you refer to, and how do we do that?  About to PR a tutorial series :).

**Comment by aoloe** [comment]
i'm referring to the redacted content that will be in this repository.

gpl, which is the license mentioned in the license file at the root of this repository, has been created for source code and is not optimal for tutorials.

i would suggest using a creative commons license, my preferred being cc-by-sa (which somehow matches the GPL...).

**Comment by luzpaz** [comment]
I agree with @aoloe

**Comment by OceanWolf** [comment]
Gah, I don't want to open a can of worms, but how does this license compare?  http://opencontent.org/openpub/  I refer to it because on the official scribus wiki, the tutorials there use that license and I would like to copy over and update some of those tutorials from there onto here...

**Comment by aoloe** [comment]
do you mean the license used by the official documentation?

in that case, don't copy the content from that part of the wiki, please.

the "official" part of the wiki is under a license (sorry, i don't know its name and too lazy right now to check) with an extension that limits the (professional) printing of it.
this makes it non free for many people and many linux distrubutions.

the rest of the wiki is cc-by-sa, the same license i propose for this repository.
the cc license is mentioned in the footer of each page of the wiki.

**Comment by OceanWolf** [comment]
Yes, at least that appeared at least that licence appeared at the bottom of some pages, as it had tha name "open publications" it sounded more like our kind of thing, I gave it a quick skim through and didn't see anything preventing us from copying/modifying it, but okay, I just thought I should mention it.

**Comment by aoloe** [comment]
this has been discussed lot of times (iirc last time was the past week).

the team sticks to their choice for the official license.
debian, ubuntu and many other linux distrubutions do not install the
scribus documentation by default (it's in the non-free repositories).

that license is not free as defined for free software so please
refrain from copying pieces of it in a documentation which does not have
exactly the same license: afaik, it's basically not compatible with any
of the free content licenses.

**Comment by luzpaz** [comment]
How do you suggest we display the license link on http://scribus.io/scribus-tutorials ?
A link at the bottom of each page displayed ?

**Comment by OceanWolf** [comment]
Yes, we could do that quite easily, a page does not need to appear in the contents for it to load, we just add it as a link either in the raw html, or the generated html from the javascript.

**Comment by aoloe** [comment]
yep, putting it in the footer is probably a good idea :-)

**Comment by OceanWolf** [comment]
@aoloe done in #15 

But I wonder about how we deal with markdown license... I use showdown to convert the markdown into html, the license for which you can find here https://github.com/showdownjs/showdown/blob/master/license.txt

Any thoughts?

**Comment by aoloe** [comment]
as long as the showdown license is free, it's not a big deal.
if we have a link to the full sources behind the manual it could be enough.
or we could create a page with the list of tools we are using and their license.

anyway: this is something that is generally not required (you should not strip the license, but it's not needed to show it in your final product).
it's like creating a book with scribus: it's not needed to write in the book that you used scribus... but it's nice to see it noted in there :-)
